packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -305,8 +305,6 @@ export type CallCompositeIcons = {
     ErrorBarClear?: JSX.Element;
     HorizontalGalleryLeftButton?: JSX.Element;
     HorizontalGalleryRightButton?: JSX.Element;
-    LobbyScreenConnectingToCall?: JSX.Element;
-    LobbyScreenWaitingToBeAdmitted?: JSX.Element;
```

Comment:
Are we okay with these deletes to the API? this would be a breaking change to anyone who overrode that icon. Should we maybe update these icons to be spinners?

Reply:
@prabhjot-msft  can we bring this up in design office hours?

Reply:
Is this something we want to support if they already started customizing this area should we just let them? We are just changing the default behavior?

Reply:
After design discussion, replaced the coffee icon with spinner